dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedFileReader.java

```diff
@@ -124,7 +124,7 @@ public CompositeDataBuffer getMultipleDataFileChannel(Channel channel, long leng
       DataBuffer dataBuffer;
       Optional<DataFileChannel> dataFileChannel =
           mPositionReader.getDataFileChannel(mPos, (int) lengthPerOp);
-      if (dataFileChannel.isEmpty()) {
+      if (!dataFileChannel.isPresent()) {
```

Comment:
What is the difference between `dataFileChannel.isEmpty()` and  `!dataFileChannel.isPresent()` ?

Reply:
Is it just because this is an old version of the code?